src/App.vue

```diff
@@ -36,6 +43,8 @@
 </template>
 
 <script>
+import 'sfgov-pattern-lab/dist/uswds/js/uswds.js'
```

Comment:
what's this?

Reply:
This is one of the JS files from the pattern library (which just pulls in uswds as a dependency).

I do think it'd be better to have the pattern library recompile into a since minified file, but it doesn't currently do that.